en/sql/identifier.rst

```diff
@@ -114,13 +132,17 @@ The following table summarizes the maximum byte length allowable for each identi
 +-----------------------+-------------------+
 | Java Stored Procedure | 254               |
 +-----------------------+-------------------+
-| Trigger               | 254               |
+| Trigger               | 222               |
 +-----------------------+-------------------+
-| View                  | 254               |
+| View                  | 222               |
 +-----------------------+-------------------+
-| Serial                | 254               |
+| Serial                | 222               |
 +-----------------------+-------------------+
 
+.. note::
+
+    Prior to version 11.2, the maximum length of table, trigger, and serial names was 254 bytes. Since version 11.2, table, trigger, and serial names contain schema names. Therefore, identifier names cannot exceed 222 bytes except for 31 bytes, which is the length of the owner name, and 1 byte, which is the length of the delimiter (.).
```

Comment:
```suggestion
    Prior to version 11.2, the maximum length of table, trigger, and serial names were 254 bytes. Since version 11.2, table, trigger, and serial names contain schema names. Therefore, identifier names cannot exceed 222 bytes except for 31 bytes, which is the length of the owner name, and 1 byte, which is the length of the delimiter (.).
```

Reply:
Thanks, I'll change it.